CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 ## NEXT VERSION
 
-## 20230522 (latest)
+## 20230611 (latest)
+* **GCC version: 9.3.0**
+* devel: add colorama package
+* phoenix-rtos-project commit
+  * devel image: `7496f4a`
+  * build image: `e8b8ba5`
```

Comment:
I'd not split it and just write `phoenix-rtos-project commit: e8b8ba5`, which relates to a version of project, where build image have been built. @nalajcie do you approve such approach? A version doesn't matter for `devel` image in my opinion.

Reply:
It matters, we have `hostutils` in **devel** image version (the `libphoenix`/`toolchain` is in **build** i image version

Reply:
Yeah that's right, it's worth it. I meant it does not matter for this specific change (adding colorama)